src/passes/Asyncify.cpp

```diff
@@ -716,13 +716,13 @@ class ModuleAnalyzer {
                             return !info.isBottomMostRuntime &&
                                    !info.inRemoveList;
                           },
-                          [verbose](Info& info, Function* reason) {
-                            if (verbose && !info.canChangeState) {
+                          [](Info& info) { info.canChangeState = true; },
+                          [verbose](const Info& info, Function* reason) {
+                            if (verbose && !info.isBottomMostRuntime && !info.inRemoveList) {
```

Comment:
I was thinking we still probably don't want to log for files in the remove list (and whatever bottom runtime means). This way when you add a function to the remove list you can clearly see its effect.

Reply:
I see now, thanks. Makes sense.

Reply:
Thinking some more, as you said, we have that code right above in `canHaveProperty`, so we can actually simplify the logic in the header to avoid this duplication: we can call `logVisit` only after checking `canHaveProperty`. That is, I am suggesting the header do this:
```cpp
if (!canHaveProperty) continue;
logVisit();
if (!hasProperty) {
  add it etc
```

Reply:
Yep, that makes sense.